user.md

```diff
@@ -1,32 +1,229 @@
+|`stateId`|The State ID correlating to facility locations (ex: `12`)|OPTIONAL|
+|`timeframe`|Time to retrieve results up until. (format: `yyyymmdd`)|OPTIONAL|
+|`statusId`|The ID of the status. (ex: `101` for "unaffected")|OPTIONAL|
+
+#### Sample response body
+
+A successful request will respond with a `200` HTTP status code and the following payload format:
+
+```
 {
-  "facilityNpi": 1003906488,
-  "patientIdFromFacility": "<some_value>",
-  "disasterId": 1001,
-  "date": "2022-08-11T18:36:39.778Z",
-  "statusId": 102
+  "unaffected": 0,
+  "injured": 0,
+  "illInFacility": 0,
+  "illNotInFacility": 0,
+  "deceased": 0,
+  "isolated": 0
 }
 ```
-7. The payload should return a status 201 with the id. If the call is unauthorized, review steps 3-5.
-8. To retrieve a Patient Status Update perform steps 3 and 4 with the EHR or GOVT role. Both EHR, GOVT roles users have permissions to get a record. Other users are unauthorized.
-9. Invoke GET /patientStatus/ using request parameters that match those used in the POST request
-*Note:* the facilityNpi and patientId must match what was passed on the post
-```json
+
+If the response returns an error, make sure you are logged in with an unexpired JWT token.
+
+### Using the patient-disaster-status-controller
+
+#### GET /patientStatus/
+
+Returns a patient's ID (not the facility's ID), and information about their most recently recorded update.
+
+**Only EHR Maintainers with a matching facilityNpi and Government Care System Coordinators have permission to receive information from this endpoint.**
+
+#### Sample request parameters
+
+|GET Parameter|Value|Optional?|
+|---|---|---|
+|`facilityNpi`|The facility's NPI (ex: `1003906488`)|REQUIRED|
+|`patientIdFromFacility`|The ID attached to a patient at the facility. (ex: `myCustomFacilityId123`)|REQUIRED|
+
+#### Sample response body
+
+A successful request will respond with a `200` HTTP status code and the following payload format:
+
+```
 {
-   "facilityNpi": 1003906488,
-   "patientIdFromFacility": "<some_value>"
+  "date": "2022-08-11T21:59:47.507Z",
+  "patientId": "string",
+  "disasterName": "string",
+  "patientStatus": "string",
+  "facilityLocation": "string"
 }
-```
\ No newline at end of file
+```
+
+where the `patientId` is the API's internally-generated ID for the patient.
+
+If the response returns an error, make sure you are logged in as either a Government Care Systems Coordinator or an EHR Maintainer with a matching `facilityNpi` that corresponds with the user.
+
+#### POST /patientStatus/
+
+Accepts an update for a patient. Requires the facility's NPI, the facility's patient ID, the ID of the disaster impacting the patient, the ID of their current status, and the date that this update was recorded on. On success, the endpoint will return the API's generated ID for the patient.
+
+**Only EHR role users have permission to create a new patient status. Other users are unauthorized.**
+
+##### Sample request body
```

Comment:
Correctly as implemented in the code, it's not a request body, but provided as query parameters. There's an example curl request for this in the 04-test.sh that may help with updating this.

Reply:
Ah, ok, so I should update to:

##### Sample request parameters

and then put it in a table like I did for the other GET parameters?

Reply:
I think that makes sense. As long as we are clear they are query parameters and that it's a POST call, we should be good.

Should we add something like an example CURL command to help with clarity?

Reply:
I updated it to a table. I don't know that I want to add CURL commands as these instructions are based on using the SwaggerUI and I don't have instructions written for authenticating via CURL. 🤔 I don't think I could write those, and would need someone else to.